src/db-api.js

```diff
@@ -74,21 +74,19 @@ const utxoSumForAddresses = (db: Pool) => async (addresses: Array<string>)
 const txHistoryQuery = (limit: number) => `
   SELECT txs.id as "dbId", txs.hash, txs.block_no, txs.blockHash, txs.block_index as tx_ordinal, txs.time, txs.body::text from (
```

Comment:
With the removal of tx_body, does this reference also need to be removed from the SELECT 
```
txs.body::text
```

History requests generate a "column txs.body does not exist" error.

Reply:
Ugh, sorry, being hasty.

Reply:
Thank you! That fix seems to have worked. This review can be closed and we'll post anything else we find.